MASH-FRET/source/simulations/plotExample.m

```diff
@@ -305,13 +306,17 @@ function plotExample(h_fig)
         % conversion factor photons to imagecounts is only valid for N, 
         % NExpN and PGN model)
         % EDIT MH: add offset for conversion to IC
-        if strcmp(opUnits, 'electron')
-%             img = phtn2arb(img, 1); % transfer function with eta=1, as detection efficiency already applied above
-%             I_don_plot = phtn2arb(I_don_plot, 1);
-%             I_acc_plot = phtn2arb(I_acc_plot, 1);
-            img = phtn2arb(img, offset, K, eta);
-            I_don_plot = phtn2arb(I_don_plot, offset, K, eta);
-            I_acc_plot = phtn2arb(I_acc_plot, offset, K, eta);
+        % add camera offset
+        % (EC-->IC)
+        img = img + offset;
```

Comment:
This is ok. But it is the same now, as it was before your first changes. Right?

Reply:
@mca-sh: As discussed via slack, we should resolve all conflicts before merging ;-) bis commits.

Reply:
No it is different, before the offset was converted in PC and added to light signal prior adding noise